### PR TITLE
chore: Update CODEOWNERS

### PR DESCRIPTION
**Pull request by tylerbutler**
There were some missing API entries in CODEOWNERS, and both build-tools and tree were not configured properly.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -47,6 +50,7 @@ api-report/azure-client.api.md                                            @micro
 api-report/azure-service-utils.api.md                                     @microsoft/fluid-cr-api
 api-report/test-client-utils.api.md                                       @microsoft/fluid-cr-api
 api-report/fluid-framework.api.md                                         @microsoft/fluid-cr-api
+api-report/fluid-static.api.md                                            @microsoft/fluid-cr-api @microsoft/fluid-cr-framework
```
Comment (tylerbutler) [inline .github/CODEOWNERS:53]: Will revert this change. Got fluid-framework and fluid-static mixed up in my head.

**Review by scottn12 — APPROVED** [review]
LGTM!

**Comment by tylerbutler** [comment]
Thanks, @scottn12!
